src/ts/layouts/DashboardLayout.tsx

```diff
@@ -18,9 +18,15 @@ class DashboardLayout extends Component<IDashboardProps, IDashboardState> {
   }
 
   render () {
+
+    const layoutStyle = {
+      backgroundImage: `url(/assets/background/color-bg-${this.props.settings.color}.svg)`,
```

Comment:
we probably can create new LayoutContainer extends existing one and add the above styles in style component module.